package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "lint": "eslint submissions/",
     "lint:write": "eslint --fix submissions/",
-    "prettier": "prettier --write \"submissions/**\""
```

Comment:
Don't change config files.)

Reply:
Ok, but Anastasiya Mashoshyna asked me to do this while she was tuning repo verifications. In the beginning Prettier was tuned to check only *.js files.
I'll remove config files from PR.

Reply:
Done, but I have some troubles with removing package.json and package-lock.json.
Git does not see this files

Reply:
@ses3332002 Try this:
```
git checkout master package.json package-lock.json

```
This should bring the files back (and make them staged for commit)
Then you can commit with something like "undo file removal".

Reply:
Thank you, but as I understand I need this files NOT to be in this PR, so the present state is exactly what I need.
Or, may be, I am wrong?
I guess the right version of package.json for main branch I can get with git pull, can't I?

Reply:
@ses3332002  If I merge your PR files `package.json package-lock.json` will be removed from all repositories.
Run command from @AMashoshyna 's comment.

Reply:
Done